content/en/docs/test.md

```diff
@@ -65,10 +65,10 @@ culpa qui officia deserunt mollit anim id est laborum.
   - To make a sub-item, indent two spaces.
     - This is a sub-sub-item.
 
+<!-- separate lists -->
```

Comment:
This feels like a terrible hack. :( Why won't a second blank line work?

Reply:
Apparently it is part of the [markdown spec](https://spec.commonmark.org/0.27/#example-269), and apparently Jekyll has the two blank lines as a convenience.